test/code/providers/Container_HostInventory_Class_Provider_UnitTest.cpp

```diff
@@ -0,0 +1,85 @@
+#include <set>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <unistd.h>
+#include <uuid/uuid.h>
+#include <vector>
+
+#include <scxcorelib/scxcmn.h>
+#include <scxcorelib/scxprocess.h>
+#include <scxcorelib/stringaid.h>
+#include <testutils/scxunit.h>
+#include <testutils/providertestutils.h>
+
+#include "Container_HostInventory_Class_Provider.h"
+#include "cjson/cJSON.h"
+
+using namespace std;
+using namespace SCXCoreLib;
+
+class ContainerHostInventoryTest : public CppUnit::TestFixture
+{
+    CPPUNIT_TEST_SUITE(ContainerHostInventoryTest);
+    CPPUNIT_TEST(TestEnumerateInstances);
+    CPPUNIT_TEST_SUITE_END();
+
+public:
+    vector<string> containers;
+
+    static string NewGuid()
```

Comment:
Who calls this? Is this stale code? Same for RunCommand, who calls that? I don't see any callers for that in TestEnumerateInstances, please cut/paste judiciously!